app/services/theme_settings_migrations_runner.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+class ThemeSettingsMigrationsRunner
+  Migration =
+    Struct.new(:version, :name, :code, :theme_field_id) do
+      def original_name
+        "#{version.to_s.rjust(4, "0")}-#{name}"
+      end
```

Comment:
I'm not sure if we need this extra method here since we can just store the original name which we have access to when creating this struct.

Reply:
I've replaced this method with a struct attribute.